R/input-checkboxgroup.R

```diff
@@ -70,6 +70,9 @@
 checkboxGroupInput <- function(inputId, label, choices = NULL, selected = NULL,
   inline = FALSE, width = NULL, choiceNames = NULL, choiceValues = NULL) {
 
+  # keep backward compatibility with Shiny < 1.0.1 (see #1649)
+  if (all(is.null(c(choices, choiceNames, choiceValues)))) choices <- character(0)
```

Comment:
I think this will work, but not in the way you're expecting. `is.null()` isn't vectorized. However, `c(NULL, NULL)` is just `NULL`, so it will evaluate to `TRUE` if all the items are NULL. The `all()` doesn't have any effect.

It would be more correct to use `vapply` and `is.null`.